build.assets/Dockerfile-centos7-assets

```diff
@@ -36,22 +36,22 @@ RUN yum install --nogpgcheck -y \
     yum clean all
 
 # Use just created devtool image with newer GCC and Cmake
-FROM --platform=$BUILDPLATFORM centos-devtoolset as clang12
+FROM --platform=$BUILDPLATFORM centos-devtoolset as clang14
 
 ARG DEVTOOLSET
 
-# Compile Clang 12.0.0 from source. It is needed to create BoringSSL and BPF files.
+# Compile Clang 14.0.6 from source. It is needed to create BoringSSL and BPF files.
 # CentOS 7 doesn't provide it as a package unfortunately.
 # This version of Clang is explicitly required for FIPS compliance when building BoringSSL.
 # For more information please refer to the section 12. Guidance and Secure Operation of:
-# https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp4407.pdf
+# TODO(reed) -- fill in URL
```

Comment:
Is this TODO for later or does it need to b a part of this PR?

Reply:
There is no updated security policy available yet, so it's still TODO for some future point.